recipe/meta.yaml

```diff
@@ -26,37 +26,45 @@ outputs:
       host:
         - python
         - pip
+        - docutils >=0.14,<1.0
       run:
         - python
         - alembic >=0.8.3,<0.9
         - bleach ==2.1.2
-        - configparser >=3.5.0, <3.6.0  # [py<35]
+        - configparser >=3.5.0,<3.6.0
```

Comment:
You may want to wait for https://github.com/conda-forge/configparser-feedstock/pull/17
I'm removing `noarch` but I did not test if that is enough to fix that issue though.
(I'll investigate more tomorrow morning.)